.github/workflows/ort.yml

```diff
@@ -1,236 +1,260 @@
+            EVENT_NAME: ${{ github.event_name }}
+            HEAD_REF: ${{ github.head_ref }}
+
         steps:
-            - name: Set the release version
-              shell: bash
-              run: |
-                  export version=`if [ "$EVENT_NAME" == 'schedule' ] || [ "$EVENT_NAME" == 'pull_request' ]; then echo '255.255.255'; else echo "$INPUT_VERSION"; fi`
-                  echo "RELEASE_VERSION=${version}" >> $GITHUB_ENV
-              env:
-                  EVENT_NAME: ${{ github.event_name }}
-                  INPUT_VERSION: ${{ github.event.inputs.version }}
-
-            - name: Set the base branch
-              run: |
-                  export BASE_BRANCH=`if [ "$EVENT_NAME" == 'schedule' ]; then echo 'main'; elif [ "$EVENT_NAME" == 'workflow_dispatch' ]; then echo "$INPUT_BRANCH"; else echo ""; fi`
-                  echo "Base branch is: ${BASE_BRANCH}"
-                  echo "BASE_BRANCH=${BASE_BRANCH}" >> $GITHUB_ENV
-              env:
-                  EVENT_NAME: ${{ github.event_name }}
-                  INPUT_BRANCH: ${{ github.event.inputs.branch }}
-
-            - name: Checkout
-              uses: actions/checkout@v4
-              with:
-                  submodules: "true"
-                  ref: ${{ env.BASE_BRANCH }}
-
-            - name: Set up JDK 11 for the ORT package
-              uses: actions/setup-java@v4
-              with:
-                  distribution: "temurin"
-                  java-version: 11
-
-            - name: Cache ORT and Gradle packages
-              uses: actions/cache@v4
-              id: cache-ort
-              with:
-                  path: |
-                      ./ort
-                      ~/.gradle/caches
-                      ~/.gradle/wrapper
-                  key: ${{ runner.os }}-ort
-
-            - name: Checkout ORT Repository
-              if: steps.cache-ort.outputs.cache-hit != 'true'
-              uses: actions/checkout@v4
-              with:
-                  repository: "oss-review-toolkit/ort"
-                  path: "./ort"
-                  ref: "26.0.0"
-                  submodules: recursive
-
-            - name: Install Rust toolchain
-              uses: dtolnay/rust-toolchain@1.78
-
-            - name: Install ORT
-              if: steps.cache-ort.outputs.cache-hit != 'true'
-              working-directory: ./ort/
-              run: |
-                  export JAVA_OPTS="$JAVA_OPTS -Xmx8g"
-                  ./gradlew installDist
-
-            - name: Create ORT config file
-              run: |
-                  mkdir -p ~/.ort/config
-                  cat << EOF > ~/.ort/config/config.yml
-                  ort:
-                    analyzer:
-                      allowDynamicVersions: true
-                      enabledPackageManagers: [Cargo, NPM, PIP, GradleInspector]
-                  EOF
-                  cat ~/.ort/config/config.yml
+          - name: Setup target branch and commit
```

Comment:
Better, in general, all over the file, to add shell value when running commands

Reply:
default is bash for linux (ubuntu) no need to override